services-camel/src/main/java/edu/unc/lib/cdr/headers/CdrFcrepoHeaders.java

```diff
@@ -27,4 +27,6 @@
 	
 	public static final String CdrBinaryChecksum = "Checksum";
 	
+	public static final String CdrBinaryUri = "BinaryUri";
```

Comment:
Not strictly a part of this PR, but would you be willing to namespace all of these headers to the cdr?  So basically change to "CdrBinaryUri" for the value